### PR TITLE
mg7 dependency should be part of the test scope

### PR DESCRIPTION
**Pull request by eparejatobes**
Otherwise this would lead to cyclic dependencies when using it from tests in mg7.

### CODE REVIEW

**Comment by eparejatobes** [comment]
Not seeing any action here I'm doing this myself

**Comment by eparejatobes** [comment]
Blocked by ohnosequences/mg7#95

File: build.sbt [hunk 1]
```diff
@@ -27,14 +26,13 @@ libraryDependencies ++= Seq(
...
 
-
 dependencyOverrides ++= Set(
   "org.apache.httpcomponents" % "httpclient" % "4.5.1",
   "org.slf4j"                 % "slf4j-api"  % "1.7.7"
 )
 
 
-fatArtifactSettings
+addFatArtifactPublishing()
```
Comment (laughedelic) [inline build.sbt]: by the way, the argument of this method is an sbt configuration. It's `Compile` by default, but you can try to use it for `Test`

Comment (eparejatobes) [inline build.sbt]: missed this, thank you!

**Comment by eparejatobes** [comment]
What's left in here:
- [ ] add values for the S3 addresses of generated data in `src/main`

**Comment by eparejatobes** [comment]
I think I'm merging this, and fixing the S3 addresses elsewhere.

**Comment by eparejatobes** [comment]
See #43. Merging.

**Pull request merged**